x-pack/plugins/task_manager/README.md

```diff
@@ -162,6 +178,39 @@ The task runner's `run` method is expected to return a promise that resolves to
 
 Other return values will result in a warning, but the system should continue to work.
 
+### Task retries when the Task Runner fails
+If a task runner throws an error, task manager will try to rerun the task shortly after (up to the task definition's `maxAttampts`).
```

Comment:
I think `maxAttampts` -> `maxAttempts`

Reply:
I swear I reread this. The brain is so bad at this stuff.